Source/WebCore/PAL/pal/text/TextEncodingRegistry.cpp

```diff
@@ -254,8 +254,19 @@ std::unique_ptr<TextCodec> newTextCodec(const TextEncoding& encoding)
     Locker locker { encodingRegistryLock };
 
     ASSERT(textCodecMap);
+    if (!encoding.isValid()) {
+        WTFLogAlways("Trying to create new text codec with invalid (null) encoding name.");
+        return nullptr;
```

Comment:
It looks risky to me to return nullptr. From the crash trace, we see that we're decoding a CSSStyleSheet. If we return nullptr now, the caller will just return an empty String for the stylesheet content, which means that the site will not render properly. Wouldn't it make more sense to fall back to UTF-8 (and still RELEASE_LOG_ERROR)?